packages/plugins/response-cache/src/index.ts

```diff
@@ -22,6 +22,11 @@ function sessionFactoryForEnvelop({ request }: YogaInitialContext) {
   return operationIdByRequest.get(request)
 }
 
+const cacheKeyFactoryForEnvelop: BuildResponseCacheKeyFunction =
+  async function cacheKeyFactoryForEnvelop({ sessionId }) {
+    return sessionId!
```

Comment:
we should avoid this `non-null assertion` here?

Reply:
Done!